### PR TITLE
fix(app-logs): fix errors in review service

### PR DESCRIPTION
**Pull request by iberryful**
1. increase memory quota from 512M to 1024M to avoid application out of memory.

2. set initial instance count to 2 for the blue green deploy demo.

3. fix some common errors in `reviews.js`

### CODE REVIEW

**Review by skakim — APPROVED** [review]
Wait for @kheyalimitra's approval before merging.

**Review by kheyalimitra — APPROVED** [review]

**Pull request merged**
